metadata/verbose_item_names.py

```diff
@@ -567,8 +568,8 @@
         "StarPiece4D",
         "StarPiece4E",
         "StarPiece4F",
-        "StarPiece50",
         "StarPiece51",
+        "StarPiece50",
```

Comment:
Why is the order inverted?

Reply:
`StarPiece50` is unused in the base mod, but `StarPiece51` is not unused. So 50 never enters the seed generator's item pool. This caused issues with plando. This swap here was a quicker fix than doing it properly in the base mod